### PR TITLE
chore: polish kb83-93

### PR DESCRIPTION
**Pull request by Meng-Heng**

### CODE REVIEW

File: knowledge-base/kb0086.md [hunk 1]
```diff
@@ -1,17 +1,13 @@
...
+### **NOTE**: This archived documentation has not been updated recently and may contain information that is no longer relevant
+
+Microsoft Word has a feature that automatically converts straight quotation marks <kbd>'</kbd> or <kbd>"</kbd> to curly quotation marks (also known as smart quotes or typographer's quotes), smart single quotation marks <kbd>‘</kbd> or smart double quotation marks <kbd>“</kbd> as you type.
+
+However, some Keyman keyboards such as EWTS Tibetan rely on the straight quote as a deadkey or in a rule match, so when it is changed to the smart quote, the deadkey or rule match no longer runs!
+
+You can disable this feature in Microsoft Word by following the instructions here:
+
+To turn this feature on or off:
+- 1) On the Tools menu, click AutoCorrect Options, and then click the AutoFormat As You Type tab.
```
Comment (darcywong00) [inline knowledge-base/kb0086.md:12]: Maybe just use numbered list instead of bullets + "1)"
e.g.
1.  On the Tools menu...
2.  Under Replace...

**Review by darcywong00 — APPROVED** [review]
lgtm

File: knowledge-base/kb0086.md [hunk 2]
```diff
@@ -1,17 +1,13 @@
...
+- 2) Under Replace as you type, select or clear the "Straight quotes" with "smart quotes" check box.
```
Comment (Meng-Heng) [inline knowledge-base/kb0086.md:13]:
```suggestion
- 1. On the Tools menu, click AutoCorrect Options, and then click the AutoFormat As You Type tab.
- 2. Under Replace as you type, select or clear the "Straight quotes" with "smart quotes" check box.
```

**Review by darcywong00 — APPROVED** [review]
lgtm

**Pull request merged**